Dockerfile

```diff
@@ -1,7 +1,4 @@
-FROM python:3.8-slim
-
-# Install dependencies for OpenCV
-RUN apt-get update && apt-get -y install libgl1 libglib2.0-0 git
```

Comment:
Unfortunately we do still need to install git in order to utilize the version command. This line can be put back in, just without libg, only git.